modules/core/server/views/layout.server.view.html

```diff
@@ -38,18 +38,21 @@
 </head>
 
 <body class="ng-cloak">
-  <header data-ng-include="'/modules/core/views/header.client.view.html'" class="navbar navbar-fixed-top navbar-inverse"></header>
-  <!--Found in index.server.view.html-->
-  <section class="banner">
-    <section class="container">
+  <div class="container">
```

Comment:
!fix I see what you are going for by killing two birds here with wrapping all the views in a container, but it might be best if we just let the containers be declared above the views they will be needed for here. We might not WANT the same padding on the sidebar that we will get on the page content views or the headers, so probably best to keep this separate for now.